man/tpm2_pcrlist.1.md

```diff
@@ -0,0 +1,72 @@
+tpm2_pcrlist 1 "AUGUST 2017" tpm2-tools
+==================================================
+
+NAME
+----
+
+tpm2_pcrlist(1) - List PCR values.
+
+SYNOPSIS
+--------
+
+`tpm2_pcrlist` [OPTIONS]
+
+DESCRIPTION
+-----------
+
+tpm2_pcrlist(1) Displays PCR values.
+
+OPTIONS
+-------
+
+  * `-g`, `--algorithm`=_HASH\_ALGORITHM_:
+    Only output PCR banks with the given algorithm.
+
+  * `-L`, `--selList`=_PCR\_SELECTION\_LIST_:
+    Only output the following PCRs ids. (0~23) for each bank.
+
+  * `-s`, `--algs`:
+    Output the list of supported algorithms.
+
+[common options](common/options.md)
+
+[common tcti options](common/tcti.md)
+
+EXAMPLES
+--------
+
+display all PCR values:
+
+```
+tpm2_pcrlist
+```
+
+Display the PCR values with a specified bank:
+
+```
+tpm2_pcrlist -g 0x04
```

Comment:
Maybe using pretty names in the examples to make them more clear?

Reply:
Done